app/views/wizard/steps/duty/gb_to_ni/_no_duty.html.erb

```diff
@@ -27,5 +27,5 @@
     <li>You are transporting goods from England, Scotland or Wales to Northern Ireland</li>
     <li>You are able to take advantage of the preferential tariffs provided by the UK / EU Trade and Co-operation Agreement (TCA) and have a valid Certificate of Origin</li>
   </ul>
-  <%= render('wizard/steps/duty/gb_to_ni/shared/note') %>
+  <p class="govuk-body-m">You may be called upon to provide a copy of your Certificate of Origin to avoid paying duties.<p>
```

Comment:
I'm going to guess that when this partial is rendered it implicitly means `certificate == true && uk_trader_scheme == false`?

Reply:
Yep